Update scalar DL configuration guide (#163)
scalar-labs/scalar-kubernetes: docs/ConfigureScalarDL.md

    # username=<AWS_ACCESS_KEY_ID>
    # password=<AWS_ACCESS_SECRET_KEY>

> # AWS RDS Databases(MySQL PostgreSQL, Oracle, SQL Server, Aurora), Azure Databases for MySQL, PostgreSQL

Reviewer: I think we can use the same keyword used in schema loader for a standard
https://github.com/scalar-labs/scalardb/tree/master/schema-loader

Author: Updated changes similar to schema loader section. But used USERNAME instead of USER as it suited better(Also there were already some discrepancies with schema loader configuration with this document).

   # username=<ACCOUNT_USERNAME>
   # password=<ACCOUNT_PASSWORD>
    
kubectl create secret generic scalardl --from-literal db-username=<username> --from-literal db-password=<password>
```
**Note**:- 

> * For Azure Database for MySQL use <ACCOUNT_USERNAME@HOST_NAME> instead of <ACCOUNT_USERNAME> as `username` value.

Reviewer: Ditto

Author: Removed the notes section for now as it may be a bit confusing.

Reviewer: I think we can keep this note if it is needed.

To enable the Auditor service, you can follow the [Enable Auditor](#enable-auditor) section.

> ### JDBC Databases

Reviewer: I think we need to specify the supported JDBC database here

Author: Updated with the list of JDBC databases.

To enable the Auditor service, you can follow the [Enable Auditor](#enable-auditor) section.

> ### AWS RDS Databases(MySQL PostgreSQL, Oracle, SQL Server, Aurora), Azure Databases for MySQL, PostgreSQL

Reviewer: ```suggestion
### JDBC databases
```

Author: Updated title section as specified.

#### Requirements

> * The JDBC connection URL (`JDBC_CONNECTION_URL`) format may vary according to the cloud service provider and database you choose. Please check the official documentation and confirm the connection URL format.

Reviewer: ```suggestion
* You must configure the appropriate JDBC connection URL (JDBC_CONNECTION_URL) for each JDBC database.
```
I think we should follow the requirements and recommendations format.

> * For `Azure Database for MySQL` use <ACCOUNT_USERNAME@HOST_NAME> instead of <ACCOUNT_USERNAME> for `username` and `dbUsername` values when updating the configuration files.

Reviewer: Ditto

Author: Removed the requirements section for now.

#### Configure schema-loading-custom-values

> To create the Scalar DL schema in AWS RDS databases, you need to update the [schema-loading-custom-values.yaml](../conf/schema-loading-custom-values.yaml) file.

Reviewer: Is it only for AWS RDS?

Author: That was a mistake, updated correctly now.

**With Kubernetes secrets**

If you have created the `scalardl` Kubernetes secret, you can use the following configuration.

```yaml
schemaLoading:
  existingSecret: scalardl

  database: jdbc
  contactPoints: <JDBC_CONNECTION_URL>
```

**Without Kubernetes secrets**

If you have not created the `scalardl` Kubernetes secret, you can use the following configuration.

```yaml
schemaLoading:
  database: jdbc
  contactPoints: <JDBC_CONNECTION_URL>
  username: <ACCOUNT_USERNAME>
  password: <ACCOUNT_PASSWORD>
```

#### Configure scalardl-custom-values

> To deploy the Scalar DL Ledger on AWS RDS databases, you need to update the [scalardl-custom-values.yaml](../conf/scalardl-custom-values.yaml) file.

Reviewer: Is it only for AWS RDS?

Author: Updated.

**With Kubernetes secrets**

If you have created the `scalardl` Kubernetes secret, you can use the following configuration.

```yaml
ledger:
  existingSecret: scalardl
  ....
  scalarLedgerConfiguration:
    dbStorage: jdbc
    dbContactPoints: <JDBC_CONNECTION_URL>
```

**Without Kubernetes secrets**

If you have not created the `scalardl` Kubernetes secret, you can use the following configuration.

```yaml
ledger:
  ....
  scalarLedgerConfiguration:
    dbStorage: jdbc
    dbContactPoints: <JDBC_CONNECTION_URL>
    dbUsername: <ACCOUNT_USERNAME>
    dbPassword: <ACCOUNT_PASSWORD>
```

To enable the Auditor service, you can follow the [Enable Auditor](#enable-auditor) section.

#### Configure scalardl-audit-custom-values

> To deploy the Scalar DL Auditor on any AWS RDS databases, you need to update the [scalardl-audit-custom-values.yaml](../conf/scalardl-audit-custom-values.yaml) file.

Reviewer: Ditto
